Core/WKWebViewExtension.swift

```diff
@@ -67,6 +67,7 @@ extension WKWebView {
         dataStore.fetchDataRecords(ofTypes: allData) { records in
             let externalRecords = records.filter { $0.displayName != Constants.cacheKey }
```

Comment:
This is what I was saying could probably become a method of some sort to take in a non const.